File: src/main/java/gargoyle/ct/ui/impl/control/CTConfigMenuItem.java
package gargoyle.ct.ui.impl.control;

import javax.swing.*;

public final class CTConfigMenuItem extends JCheckBoxMenuItem {
    private static final long serialVersionUID = -2199156620390967976L;

    @SuppressWarnings("TypeMayBeWeakened")
    public CTConfigMenuItem(CTConfigAction action) {
        super(action);
    }
}

File: src/main/kotlin/gargoyle/ct/ui/impl/control/CTConfigMenuItem.kt
package gargoyle.ct.ui.impl.control

import java.io.Serial
import javax.swing.JCheckBoxMenuItem

class CTConfigMenuItem(action: CTConfigAction?) : JCheckBoxMenuItem(action) {
    companion object {
        @Serial
        private val serialVersionUID = -2199156620390967976L
    }
}
